view/frontend/web/template/payment/card-vault-form.html

```diff
@@ -0,0 +1,91 @@
+            <img data-bind="attr: {
+            'src': getIcons(getCardType()).url,
+            'width': getIcons(getCardType()).width,
+            'height': getIcons(getCardType()).height,
+            'alt': getIcons(getCardType()).title
+            }" class="payment-icon">
+            <span translate="'ending'"></span>
+            <span text="getMaskedCard()"></span>
+            (
+            <span translate="'expires'"></span>:
+            <span text="getExpirationDate()"></span>
+            )
+        </label>
+    </div>
+
+    <div class="payment-method-content">
+
+        <!-- ko ifnot: (getClientKey()) -->
+        <span class="message message-error error"><!-- ko text: $t('Please configure an API Key, a Client Key and a live endpoint prefix(if in Production Mode) in your Adyen Required Settings')--><!-- /ko --></span>
+        <!--/ko-->
+
+        <each args="getRegion('messages')" render=""></each>
+        <div class="payment-method-billing-address">
+            <each args="data: $parent.getRegion(getBillingAddressFormName()), as: '$item'">
+                <render args="$item.getTemplate()"/>
+            </each>
+        </div>
+
+        <form class="form" action="#" method="post" data-bind="attr: {'id': getId(), 'data-role': getId() }">
+            <fieldset
+                    data-bind="attr: {class: 'fieldset payment items ccard adyen_cc_vault', id: 'payment_form_' + 'adyen_cc_form' + '_' + getId()}">
+
+                <div id="card_modal_wrapper">
+                    <div id="card_action_modal">
+                        <div id="card_action_modal_content"></div>
+                    </div>
+                </div>
+
+                <div class="field number">
+                    <label class="label">
+                        <span><!-- ko text: $t('Credit Card Number')--><!-- /ko --></span>
+                    </label>
+                    <div class="control">
+                        <span data-bind="text: getMaskedCard()"></span>
+                    </div>
+                </div>
+
+                <div class="checkout-component-dock" afterRender="renderCardVaultToken()" data-bind="attr: { id: 'cvcContainer-' + getId()}"></div>
+            </fieldset>
+        </form>
+        <div class="checkout-agreements-block">
+            <!-- ko foreach: $parent.getRegion('before-place-order') -->
+            <!-- ko template: getTemplate() --><!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="actions-toolbar">
+            <div class="primary">
+                <button class="action primary checkout"
+                        type="submit"
+                        data-bind="
+                            click: placeOrder,
+                            attr: {title: $t('Place Order')},
+                            enable: isButtonActive()
+                        "
+                        disabled>
+                    <span translate="'Place Orderrrrrr'"></span>
```

Comment:
Should we change the span's translate attribute?

Reply:
I left it as is for now to create a small difference from the normal template. Eventually this will def be fixed.

Reply:
Ah cool, thanks for the explanation.